src/test/resources/testng_suites/web-localization.xml

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite verbose="1" name="Carina Demo Tests - Web Localization Sample" parallel="methods">
+
+    <parameter name="suiteOwner" value="qpsdemo"/>
+    <parameter name="zafira_project" value="UNKNOWN"/>
+    <parameter name="jenkinsJobName" value="Web-Localization-Demo-Test"/>
+    <parameter name="jenkinsJobType" value="web"/>
+    <parameter name="jenkinsEnvironments" value="DEMO"/>
+    <parameter name="jenkinsEmail" value="demo@qaprosoft.com"/>
+
+    <parameter name="jenkinsRegressionPipeline" value="web_regression, full_regression"/>
+    <parameter name="jenkinsPipelineEnvironments" value="DEMO"/>
+
+    <parameter name="jenkinsRegressionMatrix_full_regression" value="browser: chrome; locale: en_US; locale: es_ES; locale: de_DE"/>
```

Comment:
it still declared mistakenly!
semi-colon separated: different runs of the same job in scope of regression cron launch
comma separated: number of args for each child job

based on description above, please, organize 3 child jobs run on chrome + en_US, chrome + es_ES and chrome + de_DE.

And one more item, please add example for chinese and japanese as well please